### PR TITLE
[Awesome Scala] Add explore category page

### PR DESCRIPTION
**Pull request by adpi2**
image

Result is visible here: https://index-dev.scala-lang.org/explore/graphical-interfaces-and-game-development

### CODE REVIEW

File: modules/template/src/main/twirl/scaladex/view/search/sorting.scala.html [hunk 1]
```diff
@@ -30,35 +28,12 @@
...
-      </button>
-      <button name="sort" value="forks" type="submit" 
-        class="btn btn-default @if(params.sorting.contains("forks")) { active }">
-        Forks
-      </button>
-      <button name="sort" value="contributors" type="submit"
-        class="btn btn-default @if(params.sorting.contains("contributors")) { active }">
-        Contributors
-      </button>
+      @for(sorting <- Sorting.all) {
```
Comment (mlachkar) [inline modules/template/src/main/twirl/scaladex/view/search/sorting.scala.html:31]: you can do `map` too

File: modules/template/src/main/twirl/scaladex/view/search/result.scala.html [hunk 2]
```diff
@@ -29,19 +28,19 @@ <h4 class="text-center">All the projects below have beginner-friendly issues whi
   <div class="col-md-2">
     <div class="result-count">
       @if(pagination.current == 1) {
-        @pagination.itemCount results
+        @pagination.pageSize results
       } else {
-        Page @pagination.current of @pagination.itemCount results
+        Page @pagination.current of @pagination.pageSize results
```
Comment (mlachkar) [inline modules/template/src/main/twirl/scaladex/view/search/result.scala.html:33]: so pageSize is the entire result size ?

File: modules/template/src/main/twirl/scaladex/view/explore/exploreCategory.scala.html [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  projects: Page[ProjectDocument],
+  scalaVersions: Seq[(Scala, Int)],
+  platforms: Seq[(Platform, Int)],
+  params: ExploreParams
+)
+@main(env, title = category.title, showSearch = true, user) {
+  <main id="container-explore-category">
+    <div class="explore-header">
+      <div class="container">
+        <h1><a href="/explore">Awesome Scala</a></h1>
```
Comment (mlachkar) [inline modules/template/src/main/twirl/scaladex/view/explore/exploreCategory.scala.html:25]: finally you are using `explore`

**Review by mlachkar — APPROVED** [review]
too much css ^^

File: modules/template/src/main/twirl/scaladex/view/search/result.scala.html (hunk 2 above)
Comment (adpi2) [reply modules/template/src/main/twirl/scaladex/view/search/result.scala.html:33]: no its the size of the page

File: modules/template/src/main/twirl/scaladex/view/explore/exploreCategory.scala.html (hunk 3 above)
Comment (adpi2) [reply modules/template/src/main/twirl/scaladex/view/explore/exploreCategory.scala.html:25]: I am still using explore but I have a new PR that renames everything into "/awesome"

File: modules/template/src/main/twirl/scaladex/view/search/result.scala.html (hunk 2 above)
Comment (adpi2) [reply modules/template/src/main/twirl/scaladex/view/search/result.scala.html:33]: Usually it is 20 except if this is the last page and there is less than 20 items in it

**Pull request merged**